File: Dart/bubblesort.dart
/*Method takes in a list and returns a Sorted list. Runtime O(n^2)

List<int> bubbleSort(List<int> list){
    for (int i = 0; i < list.length; i++){
        for (int j = 0; j < list.length - 1; j++){
            if (list[j] > list[j + 1]){
                int num = list[j];
                list[j] = list[j + 1];
                list[j + 1] = num;
            }
        }
    }
  return list;
}

File: Javascript/bubbleSort.js
function bubbleSort(array){
    for(let i = 0; i < array.length; i++){
        for(let j = 0; j < array.length - 1 - i; j++){
            if(array[j+1] < array[j]){
                const aux = array[j];
                array[j] = array[j+1];
                array[j+1] = aux;
            }
        }
    }
    return array
}

const array = [5, 7, 4, 100, -1, -1000, 23, 24, 0]
console.log(bubbleSort(array))
